packages/reports/addon/templates/components/navi-collection.hbs

```diff
@@ -1,79 +1,87 @@
-{{!-- Copyright 2019, Yahoo Holdings Inc. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms. --}}
-{{#if config.filterable}}
-  {{pick-single
-    selection=filter
-    options=filterOptions
-    onUpdateSelection=(action (set this "filter" _))
-    idField="filterKey"
-    displayField="name"
-  }}
-{{/if}}
-<table class="table">
-  <thead>
-    <tr>
-      <th class="align left">Name</th>
-      {{#if config.actions}}
-        <th class="align left navi-collection__actions"></th>
-      {{/if}}
-      <th class="align left">Author</th>
-      <th class="align left">Last Updated</th>
-    </tr>
-  </thead>
-  <tbody>
-    {{#unless hasTableLoaded}}
-      <tr class="is-loading">
-        <td colspan="4">
-          {{navi-loader}}
-        </td>
+{{!-- Copyright 2020, Yahoo Holdings Inc. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms. --}}
+<div class="navi-collection" ...attributes>
```

Comment:
Should we use `dir-table` from the directory package instead? These tables are very similar, we could probably get rid of this

Reply:
Directory currently depends on reports already, so we would need to move the table out of directory to avoid a circular dependency. I'm not against that, but it seems like more work than is necessary for something that isn't necessarily a problem.

Reply:
Let's just keep the scope of this PR to removing ember-pick usage. I'll make an issue for condensing the navi-collection and dir-table components into one.